Tasks/VsTest/vstest.ts

```diff
@@ -172,6 +183,15 @@ function getVstestArguments(settingsFile: string, tiaEnabled: boolean): string[]
     return argsArray;
 }
 
+function isDebugEnabled(): boolean {
```

Comment:
>isDebugEnabled [](start = 9, length = 14)

I am not sure why are we pushing /diag if system.debug=true

If these diag files are not published, it's not useful